react/ProductSummaryReference.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import type { CssHandlesTypes } from 'vtex.css-handles'
+import { useCssHandles } from 'vtex.css-handles'
+import { ProductSummaryContext } from 'vtex.product-summary-context'
+
+const { useProductSummary } = ProductSummaryContext
+
+const CSS_HANDLES = ['referenceContainer'] as const
+
+interface Props {
+  classes: CssHandlesTypes.CustomClasses<typeof CSS_HANDLES>
+}
+
+const ProductSummaryReference = ({ classes }: Props) => {
+  const { product } = useProductSummary()
+  const { handles } = useCssHandles(CSS_HANDLES, { classes })
+  const productReference: string = product.productReference ?? ''
```

Comment:
Why would you need to declare the type here? Why not just let it be undefined?

```suggestion
  const productReference = product.productReference
```

Reply:
sorry, i used this when i hasn't property in context, i forgot remove it. fixed